### PR TITLE
Fix Coachmark focus when tabbing

### LINKED ISSUE #5960: Coachmark: Prevents other components from receiving focus
### Bug Report 

- __Package version(s)__: 6.49.4 
- __Browser and OS versions__: n/a
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) High
 
Products/sites affected: (if applicable)
 
#### Describe the issue:
 
When Coachmark appears, even when collapsed, it prevents any other component on the page from getting focus. 
 
#### Actual behavior:
 
Coachmark is preventing other components from receiving focus.
 
#### Expected behavior:
 
Should not prevent focus on other components on page.
 
### If applicable, please provide a codepen repro:

Reproduces on Fabric page:
https://developer.microsoft.com/en-us/fabric#/components/coachmark

### PR DESCRIPTION
**Pull request by leddie24**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #5960
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Adds another container to Coachmark and sets its `display` to `block` or `none` depending on the Coachmark state.  Setting to `none` will prevent focus to the Primary/Secondary buttons inside of TeachingBubbleContent, so tabbing will not be trapped inside of TeachingBubbleContent.  

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx [hunk 1]
```diff
@@ -547,6 +556,7 @@ export class Coachmark extends BaseComponent<ICoachmarkProps, ICoachmarkState> i
       this._entityInnerHostElement.current.addEventListener(
         'transitionend',
         (): void => {
+          console.log('testing here');
```
Comment (JasonGore) [inline packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx]: I'm guessing you don't want this merged

**Comment by micahgodbolt** [comment]
is there a way to repro this fix?

**Comment by micahgodbolt** [comment]
@leddie24 and @JasonGore is this PR ready to be merged? What are good steps to repro this working?

**Comment by KevinTCoughlin** [comment]
> 
> 
> @leddie24 and @JasonGore is this PR ready to be merged? What are good steps to repro this working?

Pinging @leddie24 for follow-up here so we can get this fix merged

**Comment by leddie24** [comment]
Hey @KevinTCoughlin @micahgodbolt , sorry for the late response.  This PR should be ready to be merged.  I think the repro action is to open coachmark, and then press tab a few times.  The focus should be on a visible element on the page.  Previously, it was on an element in the TeachingBubbleContent, which was hidden.  This fix changes the `display` to `none` so that the elements inside of TeachingBubble can't be focused.

**Pull request merged**